app/models/miq_ae_class_copy.rb

```diff
@@ -79,13 +79,17 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
+    $miq_ae_logger.info("Destination class: #{dest_class}")
     if dest_class
+      $miq_ae_logger.info("Overwrite flag: #{@overwrite}")
       dest_class.destroy if @overwrite
+      $miq_ae_logger.info("Should only print if destination class exists and got destroyed by overwrite")
```

Comment:
@d-m-u Should this log message be conditioned by "if @overwrite", since it states the destination class exists and got destroyed by overwrite, or maybe change the wording?